src/components/forms/file-dropper/file-list-item.tsx

```diff
@@ -7,7 +7,7 @@ import { Dropdown, DropdownOption } from '../dropdown/dropdown';
 
 interface FileListItemProps {
   fileName: string;
-  fileCategories?: string[];
+  fileCategories?: { label: string; value: string }[];
```

Comment:
That's the change required to make it translatable isn't it ?

Reply:
yes